broker/lib/tags.py

```diff
@@ -1,11 +1,85 @@
-def add_tag(tags, tag_key: str, tag_value: str):
+import typing
+
+from broker.extensions import config
+from broker.lib.cf import CFAPIClient
+
+from enum import Enum
+from openbrokerapi.service_broker import (
+    ProvisionDetails,
+    ServicePlan,
+    Service,
+)
+
+cf_api_client = CFAPIClient()
```

Comment:
my gut says this belongs in `extensions` to head off potential circular dependencies in the future

Reply:
I don't think I can put this in `extensions`, because `cf.py` which contains `CFAPIClient` itself loads `config` from `extensions`. So trying to load `CFAPIClient` from `extensions` would create a circular dependency on itself